go/oasis-node/cmd/common/grpc/grpc.go

```diff
@@ -73,10 +78,15 @@ func NewServerLocal(installWrapper bool) (*cmnGrpc.Server, error) {
 func NewClient(cmd *cobra.Command) (*grpc.ClientConn, error) {
 	addr, _ := cmd.Flags().GetString(CfgAddress)
 
+	_, err := os.Stat(addr)
+	if !os.IsNotExist(err) {
```

Comment:
Does this now assume that the argument here is a socket and not an url? is this ok?

Reply:
This was cleaned now. Please recheck.